Firestore/core/test/firebase/firestore/local/remote_document_cache_test.mm

```diff
@@ -147,7 +147,7 @@
 TEST_P(RemoteDocumentCacheTest, SetAndReadDeletedDocument) {
   persistence_->Run("test_set_and_read_deleted_document", [&] {
     absl::optional<MaybeDocument> deleted_doc = DeletedDoc(kDocPath, kVersion);
-    cache_->Add(*deleted_doc);
+    cache_->Add(*deleted_doc, deleted_doc->version());
```

Comment:
Changes here and below are to make the new RemoteDocumentCache unit tests compile in the branch.